assets/js/_main.js

```diff
@@ -132,26 +132,19 @@ $(document).ready(function(){
   // the value is the key to the displayed route in the device_common/old/new variable below
   // 
   var methods = {
```

Comment:
You have to match the methods to the later paths

Reply:
Resolved. (I think.)